src/helix/impl/props.cljc

```diff
@@ -95,6 +95,9 @@
                                   ;; React Native allows arrays of styles
                                   (into-js-array (map primitive-obj v))
                                   (primitive-obj v)))
+                :value (set-obj o "value" `(if (nil? ~v)
+                                             ""
+                                             ~v))
```

Comment:
I think that this doesn't work for the `.cljs` runtime branch. I also think it's more correct to pass in `undefined` than an empty string; we aren't sure if a string is valid as the `:value` prop always.

try this
```suggestion
                :value (set-obj o "value" #?(:clj `(if (nil? ~v)
                                                     js/undefined
                                                     ~v)
                                             :cljs (if (nil? v)
                                                     js/undefined
                                                     v))
```

We could probably extract that into a helper function `some-or-undefined`.